pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -228,14 +228,22 @@ message PodSandboxConfig {
     repeated PortMapping port_mappings = 5;
     // Key-value pairs that may be used to scope and select individual resources.
     map<string, string> labels = 6;
-    // Annotations is an unstructured key value map that may be set by external
-    // tools to store and retrieve arbitrary metadata. There are a few features are
-    // driven by annotations, Runtimes could support them optionally:
+    // Unstructured key value map that may be set by external tools to store
+    // and retrieve arbitrary metadata. Annotations MUST NOT be altered or in
+    // any way processed by the runtime.
```

Comment:
Annotation allows users (e.g., kubelet) to store metadata that only matters to that the user. Runtime will treat it as a blob and return it as it-is when user query for the object.